qtconsole/ansi_code_processor.py

```diff
@@ -185,6 +182,20 @@ def set_csi_code(self, command, params=[]):
             count = params[0] if params else 1
             self.actions.append(ScrollAction('scroll', dir, 'line', count))
 
+        elif (command == 'A'):  # Move N lines Up
+            dir = 'up'
+            count = params[0] if params else 1
+            self.actions.append(MoveAction('move', dir, 'line', count))
+        elif (command == 'B'):  # Move N lines Down
```

Comment:
```suggestion

        elif command == 'B':  # Move N lines Down
```

Follow the format used above these lines.